### DISCUSSION TITLE
Monaco/web worker support

### DISCUSSION THREAD

**Discussion by deathlynocturnal** [vuejs/language-tools#598] [General]
Hello!

I was just wondering if there are any possibilities for web support or a Monaco editor plugin?

If not, what are the issues that are preventing it, as I would really like to use this in a Monaco editor instance.

Thanks,

**Comment by johnsoncodehk** [comment]
I think the current situation is easy to do. If someone is interested, please check:
- https://github.com/yaegassy/coc-volar
- https://github.com/TypeFox/monaco-languageclient
- https://github.com/open-vsx/publish-extensions/pull/436

**Reply by deathlynocturnal** [reply]
Awesome okay, I shall have a look at this, thanks!

**Comment by KenEucker** [comment]
@johnsoncodehk are you aware of any update on support for vue in monaco-editor? The closest I've found a solution is here: https://github.com/Kingwl/monaco-volar

Which I tried but haven't been able to get to work as expected.

**Reply by johnsoncodehk** [reply]
What problems are you having with monaco-volar? Maybe @Kingwl can help.

On the other hand, I also plan to integrate Monaco into the framework, please track: https://github.com/volarjs/framework/issues/2

**Reply by KenEucker** [reply]
@johnsoncodehk the highlighting and styling isn't working. I feel like the onigasm code isn't doing what is expected from that project. I also dug a bit deeper and it seems that package is targeting something slightly different.

Love your work! I will keep checking back.
